### ISSUE TITLE
Pattern:  Content group - with Pictograms - Optional learn more links at item level in patterns

### ISSUE THREAD

**Issue by ltitle** [carbon-design-system/ibm-dotcom-library#1235]
When looking at both the List Section pattern and the Pictogram Array pattern I noticed that there is no ability in Storybook to remove the item level "Learn more ->" links. These should be optional links so the authors can choose whether or not to have them.

Similarly, when viewing the Use Case pattern I do not see any "Learn more ->" links or the ability to turn them on in Storybook.   This issue will be resolved in Storybook.

Expected outcome of implementation:
![image](https://user-images.githubusercontent.com/57198117/73383557-8b225780-4297-11ea-98af-68fc8923b93b.png)

![image](https://user-images.githubusercontent.com/57198117/73383595-9bd2cd80-4297-11ea-86f7-d7be7ae5c021.png)

![image](https://user-images.githubusercontent.com/57198117/73383620-a7be8f80-4297-11ea-9516-fd50f95b48f7.png)

![image](https://user-images.githubusercontent.com/57198117/73383638-af7e3400-4297-11ea-8416-bb7f72e5c784.png)

Acceptance criteria:
Item level text lists are optional for the following patterns:
- [ ] Content block - mixed groups
- [x] Content group - with pictograms
- [ ] Content block - with media

**Comment by photodow** [comment]
Just to reiterate based on the conversation in our meeting today.

The `ContentItem` is used throughout all of these patterns, and handles the optional CTA. So to be clear the CTAs are optional in these patterns.

It's only on the story side where we've hardcoded them with the adopter dummy data we're passing in. We could possibly add a toggle that allows these to be turned on or off or something similar so they can be played with and tested in that way using storybook.

**Comment by RaphaelAmadeu** [comment]
@oliviaflory , I was talking with @kennylam about these changes to the stories, and we concluded that as the `ContentBlock` will be refactored in order to use the `ContentGroup` and the Docs will be updated to reference the Sub-patterns capabilities on having the optional CTAs, updating the storybook now does not seem viable for the `ContentBlock`.

For the `ContentGroupPictograms`, I've updated the story adding a knob for optional CTA, and it's currently on PR state.

Thanks!
